server_admin/topics/users/delete-user.adoc

```diff
@@ -1,19 +1,14 @@
-[[_delete-user]]
-
-=== Deleting Users
-
-To delete a user click on `Users` in the left menu bar.
-
-.Users
-image:{project_images}/users.png[]
-
-This menu option brings you to the user list page. Click `View all users` or search to find the user you intend to delete.
-
-.View All Users
-image:{project_images}/delete-user.png[]
-
-In the list of users, click `Delete` next to the user you want to remove. You will be asked to confirm that you are sure you want to delete this user. Click `Delete` in the confirmation box to confirm.
+[deleting-users]
+= Deleting Users
 
+To remove a user from the console you must delete the user.
 
+.Prerequisites
+* You are logged in to the correct realm and are not in the master realm.
 
+.Procedure
+. Click on *Users* in the left menu bar. This will direct you to the user list page.  
+. Click *View all users* or use the search bar to find the user you intend to delete.
```

Comment:
Clicking view all and searching for a specific user are not analogous actions. 
"Intend" is not an appropriate term. "... find the user to delete" is a better way to say it.

Reply:
It comes up by default by viewing all users.  I suggest "Search for the user to delete."  They will see the "Search" field.